controller_manager/src/controller_manager.cpp

```diff
@@ -92,11 +92,42 @@ ControllerManager::ControllerManager(
   loader_(std::make_shared<pluginlib::ClassLoader<controller_interface::ControllerInterface>>(
     kControllerInterfaceName, kControllerInterface))
 {
-  resource_manager_->start_components();
-
   init_services();
 }
 
+void ControllerManager::init_resource_manager(const std::string & robot_description)
```

Comment:
It is a bit, but now we have to handle which components are configured and activated when starting the application. For clarity, I decided to move this in to a separate method.

I don't know if it makes sense to move this method and introduction of parameters into separate PR. This would be quite complicated to do because test should be quite changed too. Or maybe I should adapt PR name.

Reply:
yeah that's ok, was mostly curious, don't want to create more work for you